<launch>
    <arg name="namespace" default="puddles" doc="Name of the vehicle"/>

    <group ns="$(arg namespace)">
        <node pkg="riptide_gazebo" name="depth_remap" type="depth_remap.py" />
        <node pkg="riptide_gazebo" name="thrust_remap" type="thrust_remap.py">
            <param name="vehicle_config" value="$(find riptide_descriptions)/config/$(arg namespace).yaml" />
        </node>
        
        <node name="stereo_image_proc" type="stereo_image_proc" pkg="stereo_image_proc" output="screen" ns="stereo">
            <param name="approximate_sync" value="true" />
            <param name="queue_size" value="10" />
        </node>
    </group>

</launch>